src/components/form.js

```diff
@@ -36,17 +36,21 @@ const Form = () => {
   ];
 
   return (
-    <form onSubmit={handleFormSubmit}>
-      <input type="text" placeholder="Book title" value={title} onChange={handleTitleChange} required />
-      <input type="text" placeholder="Author" value={author} onChange={handleAuthorChange} required />
-      <select value={category} onChange={(e) => setCategory(e.target.value)} required>
-        <option value="" disabled>Select a category</option>
-        {categories.map((cat) => (
-          <option key={cat} value={cat}>{cat}</option>
-        ))}
-      </select>
-      <button type="submit">ADD BOOK</button>
-    </form>
+    <div className="form-section">
+      <hr />
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <form onSubmit={handleFormSubmit}>
+        <input type="text" placeholder="Book title" value={title} onChange={handleTitleChange} required />
+        <input type="text" placeholder="Author" value={author} onChange={handleAuthorChange} required />
```

Comment:
The placeholder of the `book title` and the `author` looks slightly different to the actual design as shown below. Kindly ensure you use the specified color of #c4c4c4 and nothing else to get the actual design specified. 

### Your design 

![Screenshot from 2023-09-29 19-38-24](https://github.com/Uzair-Manzoor/bookstore/assets/65233860/5b9bbe1f-0c56-43df-87a7-435e2d8d7333)


### Actual design 
![Screenshot from 2023-09-29 19-51-22](https://github.com/Uzair-Manzoor/bookstore/assets/65233860/594a0a02-505e-44b4-ad2a-411b52f82f5a)